src/device-registry/utils/create-event.js

```diff
@@ -35,6 +36,100 @@ const httpStatus = require("http-status");
 
 const createEvent = {
   getMeasurementsFromBigQuery: async (req) => {
+    try {
```

Comment:
modifications to the util which retrieves measurements from Big Query. Majority of the PR changes are here. That is, the most important changes are here.